airbyte-integrations/connectors/source-snapchat-marketing/integration_tests/abnormal_state.json

```diff
@@ -1,23 +1,23 @@
 {
```

Comment:
I do not think that in this PR you should change the files of another connector (not Chargebee)!

Reply:
Without it `format` command is failing.

Reply:
Done.